modules/server/src/topics/shortcuttopic.cpp

```diff
@@ -101,23 +101,46 @@ std::vector<nlohmann::json> ShortcutTopic::shortcutsJson() const {
     return json;
 }
 
-void ShortcutTopic::sendData() const {
-    const nlohmann::json data = { {"shortcuts", shortcutsJson()} };
-    _connection->sendJson(wrappedPayload(data));
+nlohmann::json ShortcutTopic::shortcutJson(const std::string& identifier) const {
+    std::vector<nlohmann::json> json;
+    std::vector<interaction::Action> actions = global::actionManager->actions();
+
+    auto found = std::find_if(
+        actions.begin(),
+        actions.end(),
+        [&identifier](const interaction::Action& action) {
+            return action.identifier == identifier;
+    });
+
+    if (found == actions.end()) {
+        return json;
```

Comment:
```suggestion
        return {};
```

Making it more clear that it is an empty result